less/index.less

```diff
@@ -57,3 +57,45 @@ html, body {
     height: 100%;
 }
 
+// max width for website
+@maxWidthWebsite: 800px;
+
+// base font sizes
+@defaultFontSize: 100%;
+@mobileFontSize: 150%;
+
+// font sizes
+@xLargeFontSize: 2rem;
+@largeFontSize: 1.5rem;
+@mediumFontSize: 1rem;
+@smallFontSize: .5rem;
+
+/ breakpoints
+@mobileBreakpoint: 500px;
```

Comment:
make sure your comments start with // not just / otherwise this is not valid syntax